plugins/modules/kibana_fleet_host.py

```diff
@@ -85,42 +85,17 @@
   - expedient.elastic.elastic_auth_options.documentation
 '''
 
+from ansible.module_utils.basic import _ANSIBLE_ARGS, AnsibleModule
```

Comment:
what are you using `_ANSIBLE_ARGS` for? generally want to avoid importing internal stuff like that even if there's nothing technical in python stopping you